src/Arcus.Messaging.Tests.Integration/ServiceBus/ServiceBusClient.cs

```diff
@@ -16,15 +18,15 @@ namespace Arcus.Messaging.Tests.Integration.ServiceBus
     public class ServiceBusClient
     {
         private readonly KeyRotationConfig _configuration;
-        private readonly ITestOutputHelper _logger;
+        private readonly ILogger _logger;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ServiceBusClient"/> class.
         /// </summary>
         /// <param name="configuration">The configuration instance to provide the necessary information during authentication with the correct Azure Service Bus instance.</param>
         /// <param name="logger">The instance to log diagnostic messages during the interaction with the Azure Service Bus instance.</param>
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="configuration"/> or the <paramref name="logger"/> is <c>null</c>.</exception>
-        public ServiceBusClient(KeyRotationConfig configuration, ITestOutputHelper logger)
+        public ServiceBusClient(KeyRotationConfig configuration, ILogger logger)
```

Comment:
We're building a service bus client here so I would rename the config to ServiceBusNamespaceConfiguration or so.

Entities should be left out and passed per operation

Reply:
For example, give me the keys for the orders queue or xyz subscription on topic foo

Reply:
Yes, no problem.

Reply:
Discussed somewhere else. Resolving.